tests/bruno/e2e/3-nodes-transfer/12-node1-add-tlc.bru

```diff
@@ -0,0 +1,50 @@
+meta {
```

Comment:
There are two ways to process the invoice here. The first is to make the wallet parse the invoice and send `AddTlc` to the ckb-pcn-node, the second is let the ckb-pcn-node directly process invoice. Since we already have `NewInvoice` for ckb-pcn-node, maybe we should also make ckb-pcn-node process invoices from another node.